packages/desktop-client/src/components/accounts/Account.js

```diff
@@ -369,9 +369,22 @@ function SelectedBalance({ selectedItems }) {
       .calculate({ $sum: '$amount' }),
   });
 
-  if (balance == null) {
-    return null;
+  let scheduleBalance = null;
+  let scheduleData = useCachedSchedules();
+  let previewIds = [...selectedItems]
+    .filter(id => isPreviewId(id))
+    .map(id => id.slice(8));
+  for (let s of scheduleData.schedules) {
+    if (previewIds.includes(s.id))
+      if (!account || account.id === s._account) scheduleBalance += s._amount;
+      else scheduleBalance -= s._amount;
   }
+
+  if (balance == null) {
+    if (scheduleBalance == null) return null;
+    else balance = scheduleBalance;
+  } else if (scheduleBalance != null) balance += scheduleBalance;
+
```

Comment:
Can you add braces to all of these if statements? I find it a bit hard to read this way. (Also opened #1015 to enforce this more often in the future)

Reply:
Sure thing !

I wasn't too fine with the original presentation either, I was just trying to comply to the rules of eslint.
I hope things looks ok now :)